test/test_helper.rb

```diff
@@ -77,9 +77,13 @@ def after_teardown
 #
 module PerformJob
   def perform_job(klass, *args)
-    resque_job = Resque::Job.new(:testqueue, 'class' => klass, 'args' => args)
+    resque_job = instantiate_job( klass, *args)
```

Comment:
Is this change just a refactor? I might be missing something but it doesn't look like it makes any functional change.